test/integration/server-test.js

```diff
@@ -81,6 +87,10 @@ test('server create event with non-existing branch name', t => {
       documentation_url: 'https://developer.github.com/v3/repos/#get-branch'
     })
 
+  const configure = function (yaml) {
+    return {repository: 'name', labels: ['label']}
+  }
+
   handleCreateEvent({
```

Comment:
now that `handleCreateEvent()` returns a promise, add an `await` before it. You’ll need to make the function async, too (replace `t => {` becomes `async (t) => {`